test/testdata/lsp/completion/instance_var.rb

```diff
@@ -0,0 +1,133 @@
+# typed: true
+
+class InstanceVariable
+  def some_method
+    @different_prefix = 10
+    @my_ivar = 5
+  end
+
+  def other_method
+    my = 5
+    @my
+    #  ^ completion: @my, @my_ivar
+  end
+
+  def more_method
+    @ # error: unexpected
+    #^ completion: (nothing)
+  end
```

Comment:
For the places where we wish there were completion results but it's not doing the right thing, can we leave behind comments in case someone fixes this in the future and is confused why tests are failing?